src/main/java/uk/gov/hmcts/reform/cwrdapi/client/domain/CaseWorkerProfile.java

```diff
@@ -30,15 +30,15 @@
 
     private String id;
 
-    @MappingField(columnName = "FIRST NAME")
+    @MappingField(columnName = "First Name")
     @NotEmpty
     private String firstName;
 
-    @MappingField(columnName = "LAST NAME")
+    @MappingField(columnName = "Last Name")
     @NotEmpty
     private String lastName;
 
-    @MappingField(columnName = "Official Email", position = 1)
+    @MappingField(columnName = "Email", position = 1)
     @Email(message = CaseWorkerConstants.INVALID_EMAIL)
     @Pattern(regexp = CaseWorkerConstants.USER_NAME_PATTERN + "@"
```

Comment:
Static import of the constants would be nicer

Reply:
sure, will change :)